lib/game_logic.rb

```diff
@@ -0,0 +1,101 @@
+require('rainbow')
+# rubocop:disable Style/GlobalVars,Metrics/MethodLength
+$board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+$winner = false
+
+class Drawer
+  def draw_board
+    board_sign = $board
+    puts("\t#{board_sign[0]} | #{board_sign[1]} | #{board_sign[2]} ")
+    puts "\t---------"
+    puts("\t#{board_sign[3]} | #{board_sign[4]} | #{board_sign[5]} ")
+    puts "\t---------"
+    puts("\t#{board_sign[6]} | #{board_sign[7]} | #{board_sign[8]} ")
+    puts "\n"
+  end
+
+  def winner?(sign)
+    board_sign = $board
+    arr = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]]
+    arr.each do |element|
+      $winner = element.all? { |index| board_sign[index] == sign }
+      return true if $winner
+    end
+
+    $winner
+  end
```

Comment:
This method's not actually related to the board, but to the game logic. You should put it in a new class (I suggest Game).

Also, remember that methods that finish with `?` should return a boolean value. Yours is returning `$winner` on line 26. Please change this.